internal/configload/configload.go

```diff
@@ -45,6 +45,10 @@ var DBs = map[string]string{
 
 	"ferretdb2-dev":         "mongodb://127.0.0.1:47003/",
 	"ferretdb2-dev-secured": "mongodb://username:password@127.0.0.1:47004/",
+
+	"ferretdb2-release":     "mongodb://127.0.0.1:47005/",
+	"ferretdb2-dev-release": "mongodb://127.0.0.1:47006/",
+	"ferretdb2-prod":        "mongodb://127.0.0.1:47007/",
```

Comment:
Existing ones are:
* `ferretdb2` uses production build of documentdb release, with production build of current FerretDB `main` branch
* `ferretdb-dev` uses development build of documentdb `ferretdb` branch, with development build of current FerretDB `main` branch

New ones are:
* `ferretdb2-release` tracks release of both FerretDB and documentdb
* `ferretdb2-dev-release` tracks development build of release of both FerretDB and documentdb
* `ferretdb2-prod` tracks current main/ferretdb branches of production build

We probably no longer want `ferretdb2`, but that can be removed later.

Reply:
Actually, if we change `ferretdb2` to start using release tag of FerretDB and documentdb, would it cause an issue in data?

Reply:
Yeah, let's keep `ferretdb2` and `ferretdb2-dev`, for now, at least